operator/cmd/mesh/manifest-apply.go

```diff
@@ -108,8 +108,15 @@ func InstallCmd(logOpts *log.Options) *cobra.Command {
 	return ic
 }
 
+func checkDeprecatedFlags(cmd *cobra.Command, iArgs *installArgs) {
+	if iArgs.manifestsPath == "" {
+		cmd.PrintErrln("! Using deprecated compiled in charts, which will be removed in a future release. Please pass --charts. See https://istio.io/latest/docs/setup/install/istioctl/ for more information.")
```

Comment:
I think this URL will be more appropriate?
https://istio.io/latest/docs/setup/install/istioctl/#install-from-external-charts